src/Quarrel.ViewModels/Models/Bindables/BindableChannel.cs

```diff
@@ -437,38 +438,44 @@ public bool IsTyping
             get => Typers.Count > 0;
         }
 
-        public List<string> Names
+        public List<string> GetNames()
         {
-            get
+            List<string> names = new List<string>();
+            var keys = Typers.Keys.ToList();
+
+            foreach (var id in keys)
             {
-                List<string> names = new List<string>();
-                foreach (var id in Typers.Keys)
+                var typer = _CurrentUsersService.Users.TryGetValue(id, out var user) ? user.DisplayName : null;
+
+                if (typer != null)
```

Comment:
Is there a reason why you choose to use a temporary variable instead of doing this?
if(TryGetvalue(id, out var user){
    names.add(user.DisplayName);
}

Reply:
I was debugging and needed the local variable.  You can change it.